slack_bolt/app/app.py

```diff
@@ -178,6 +180,8 @@ def message_hello(message, say):
             url_verification_enabled: False if you would like to disable the built-in middleware (Default: True).
                 `UrlVerification` is a built-in middleware that handles url_verification requests
                 that verify the endpoint for Events API in HTTP Mode requests.
+            attaching_function_token_enabled: False if you would like to disable the built-in middleware (Default: True).
+                `AttachingFunctionToken` is a built-in middleware that handles tokens with function requests from Slack.
```

Comment:
I think we should explain what "handling tokens" means in this context. Does the following suggestion make sense?
```suggestion
                `AttachingFunctionToken` is a built-in middleware that uses just-in-time workflow-execution-scoped tokens when your app receives a `function_executed` event.
```